Testreglar/4.1.2/Nett/nett-4.1.2b.json

```diff
@@ -0,0 +1,314 @@
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.4"
+				},
+				"nei": {
+					"type": "avslutt",
+					"fasit": "Nei",
+					"utfall": "Knappen er ikke kodet med role button."
+				}
+			},
+			"kilde": [
+				"ARIA4",
+				"F59",
+				"G10"
+			]
+		},
+		{
+			"stegnr": "3.4",
+			"spm": "Har knappen et tilgjengelig navn, som ikke er tomt?",
+			"ht": "<p>Bruk Accessibility Checker som beskrevet i tidligere steg</p>\n<p>Sjekk at informasjon under \"Name\" ikke er tomt.</p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.5"
+				},
+				"nei": {
+					"type": "avslutt",
+					"fasit": "Nei",
+					"utfall": "Knappen har ikke et tilgjengelig navn."
+				}
+			},
+			"kilde": [
+				"ARIA14",
+				"ARIA16",
+				"F68",
+				"G108",
+				"H44",
+				"H65",
+				"H88",
+				"H91",
+				"F86"
+			]
+		},
+		{
+			"stegnr": "3.5",
+			"spm": "Beskriver tilgjengelig navn formålet med knappen?",
+			"ht": "<p>Slik tester du:</p>\n<ul>\n<li>Ta utgangspunkt i det tilgjengelige navnet du fant i forrige steg.</li>\n<li>Sjekk om det tilgjengelige navnet beskriver formålet med skjemaelementet. Det er tilstrekkelig at det tilgjengelige navnet identifiserer skjemaelementet.</li>\n</ul>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.6"
+				},
+				"nei": {
+					"type": "avslutt",
+					"fasit": "Nei",
+					"utfall": "Knappen har et tilgjengelig navn, som ikke beskriver formålet med knappen."
+				}
+			},
+			"kilde": [
+				"ARIA14"
+			]
+		},
+		{
+			"stegnr": "3.6",
+			"spm": "Hvilket attributt gir tilgjengelig navn til knappen?",
+			"ht": "<ul>\n<li>Sjekk dette under \"Name\" under Computed Properties i Accessibility Checker</li>\n<li>Velg et alternativ.</li>\n</ul>",
+			"type": "radio",
+			"svarArray": [
+				"aria-labelledby",
+				"aria-label",
+				"label",
+				"innhold",
+				"title",
+				"alt",
+				"value",
+				"annet"
+			],
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "3.7"
+				}
+			}
+		},
+		{
+			"stegnr": "3.7",
+			"spm": "Har knappen mer enn en tilstand når brukeren samhandler med den?",
+			"ht": "<p>En tilstand er en dynamisk egenskap som endres når brukeren samhandler med knappen.</p>\n<p>Trykk på knappen og sjekk om knappen har minst en av disse tilstandene: </p>\n<ul>\n<li>av/på knapp (toggle button)</li>\n<li>spill av og stopp knapp</li>\n<li>en knapp som utvider eller sammenslutter innhold, for eksempel en meny-knapp</li>\n</ul>",
```

Comment:
Har det blitt vurdert om dette skal testes i en egen regel?

Reply:
@janBeniaminKwiek Jeg mener vi har vært inne på det, men at vi kom til at det ikke var nødvendig å splitte. Hva tenker du er fordelen(e)/behovet for å teste dette i en egen testregel?